docs/controls/file-browser/v8-schema.md

```diff
@@ -0,0 +1,437 @@
+                 * @example
+                 *  'https://contoso.sharepoint.com/path/to/web/Shared Documents'
+                 */
+                list?: string;
+                /**
+                 * Path segment to a folder within a list, or a server-relative URL to a folder.
+                 * @example
+                 *  'General'
+                 * @example
+                 *  'foo/bar'
+                 * @example
+                 *  '/path/to/web/Shared Documents/General'
+                 */
+                folder?: string;
+                /**
+                 * Auto fallback to root folder if the specified entry sub folder doesn't exist.
+                 */
+                fallbackToRoot?: boolean;
+            };
+            /**
+             * Indicates SharePoint ID values which may be used as a backup in case path-based navigation to the initial item fails.
+             * Id-based lookup in SharePoint is slower, as should only be used as a last-resort.
+             * The File Browser will return an error if only ID values are specified.
+             */
+            byId?: {
+                webId?: string;
+                listId?: string;
+                uniqueId?: string;
+            };
+        };
+        /**
+         * Indicates that the File Browser should start in the user's OneDrive.
+         */
+        oneDrive?: {
+            /**
+             * Specifies that File Browser should start in the user's Files tab.
+             */
+            files?: {
+                /**
+                 * Path segment for sub-folder within the user's OneDrive.
+                 * @example
+                 *  'Pictures'
+                 * @example
+                 *  '/personal/user_contoso_com/Documents/Attachments'
+                 */
+                folder?: string;
+                /**
+                 * Auto fallback to root folder if the specified entry sub folder doesn't exist.
+                 */
+                fallbackToRoot?: boolean;
+            };
+            /**
+             * Indicates that File Browser should start in the user's recent files.
+             */
+            recent?: {};
+            /**
+             * Indicates that File Browser should start in the files shared with the user.
+             */
+            sharedWithMe?: {};
+            /**
+             * Indicates that File Browser should start in the phone link data.
+             */
+            phoneLink?: {};
+            /**
+             * Indicates that File Browser should start in the user's photos.
+             */
+            photos?: {};
+            /**
+             * Indicates that File Browser should start in the user's lists.
+             */
+            myLists?: {};
+        };
+        /**
+         * Providing this object indicates that the host app can provide OAuth tokens
+         * via the existing messaging support.
+         */
+        authentication: {};
```

Comment:
Add
```ts
authentication: {
  claimsChallenge?: {
    enabled?: boolean;
  };
}
```
Indicate that passing at least `authentication: {}` is *required* in order for the File Browser to be able to authenticate and load in an `iframe`.